cmd/ddev/cmd/delete-images.go

```diff
@@ -86,10 +84,10 @@ func deleteDdevImages(deleteAll bool) error {
 	// Sort so that images that have -built on the end
 	// come up before their parent images that don't
 	sort.Slice(images, func(i, j int) bool {
-		if images[i].RepoTags == nil || len(images[i].RepoTags) == 0 {
+		if len(images[i].RepoTags) == 0 {
```

Comment:
S1009: should omit nil check; len() for nil slices is defined as zero (staticcheck)